### PR TITLE
Feature/wos 219 initial content for default values in bulma

### PR DESCRIPTION
**Pull request by Rafal-DS**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue) labeled with `bug`
- [ ] New feature (non-breaking change which adds functionality) labeled with `enhancement`
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the [code standards](../CONTRIBUTING.md) of this project.
- [ ] My change requires updating the documentation. I have updated the documentation accordingly.

### CODE REVIEW

File: application/backend/src/main/resources/libs/bulma/components/content/template/.content.json [hunk 1]
```diff
@@ -2,5 +2,7 @@
   "jcr:primaryType": "nt:unstructured",
   "color": "grey_has-text-grey",
   "shadeGrey": "darker",
-  "text": "<p>Content</p>"
+  "text": "<p>Content</p>",
+  "size": "is-small",
```
Comment (paweljozwicki) [inline application/backend/src/main/resources/libs/bulma/components/content/template/.content.json]:
```suggestion
  "size": "is-normal",
```

File: application/backend/src/main/resources/libs/bulma/components/button/template/.content.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "jcr:primaryType": "nt:unstructured",
+  "type": "button",
+  "size": "is-small",
```
Comment (paweljozwicki) [inline application/backend/src/main/resources/libs/bulma/components/button/template/.content.json]:
```suggestion
  "size": "is-normal",
```

File: application/backend/src/main/resources/libs/bulma/components/message/template/.content.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+  "jcr:primaryType": "nt:unstructured",
+  "size": "is-small",
```
Comment (paweljozwicki) [inline application/backend/src/main/resources/libs/bulma/components/message/template/.content.json]:
```suggestion
  "size": "is-normal",
```

File: application/backend/src/main/resources/libs/bulma/components/progressbar/template/.content.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+  "jcr:primaryType": "nt:unstructured",
+  "size": "is-small",
```
Comment (paweljozwicki) [inline application/backend/src/main/resources/libs/bulma/components/progressbar/template/.content.json]:
```suggestion
  "size": "is-normal",
```

File: application/backend/src/main/resources/libs/bulma/components/section/template/.content.json [hunk 5]
```diff
@@ -0,0 +1,4 @@
+{
+  "jcr:primaryType": "nt:unstructured",
+  "size": "is-small"
```
Comment (paweljozwicki) [inline application/backend/src/main/resources/libs/bulma/components/section/template/.content.json]:
```suggestion
  "size": "is-normal"
```

File: application/backend/src/main/resources/libs/bulma/components/tabs/template/.content.json [hunk 6]
```diff
@@ -1,5 +1,7 @@
 {
   "jcr:primaryType": "nt:unstructured",
+  "alignment": "",
+  "size": "is-small",
```
Comment (paweljozwicki) [inline application/backend/src/main/resources/libs/bulma/components/tabs/template/.content.json]:
```suggestion
  "size": "is-normal",
```

File: application/backend/src/main/resources/libs/bulma/components/tag/template/.content.json [hunk 7]
```diff
@@ -0,0 +1,7 @@
+{
+  "jcr:primaryType": "nt:unstructured",
+  "isLight": "false",
+  "isRounded": "false",
+  "size": "is-small",
```
Comment (paweljozwicki) [inline application/backend/src/main/resources/libs/bulma/components/tag/template/.content.json]:
```suggestion
  "size": "is-normal",
```

File: application/backend/src/main/resources/libs/bulma/components/button/template/.content.json [hunk 8]
```diff
@@ -0,0 +1,21 @@
...
+  "variant": "is-primary",
+  "openModal": "open-modal",
+  "selectOrTypeLeft": "select",
+  "selectOrTypeRight": "select",
+  "iconLibTypeLeft": "fas",
```
Comment (paweljozwicki) [inline application/backend/src/main/resources/libs/bulma/components/button/template/.content.json]: let's use mdi as in other componets

Comment (Rafal-DS) [reply application/backend/src/main/resources/libs/bulma/components/button/template/.content.json]: Done.

**Review by paweljozwicki — APPROVED** [review]

**Comment by martyna-ds** [comment]
Testing results:
Components which still has following error when editing in side panel:
Screenshot 2023-06-16 at 11 24 44
- progress bar - field "progress bar value"
- navbar item in "navbar" component - all fields
- button in "buttons" component - field "action",  "icon left", "icon left size" (same for icon right)

**Comment by Rafal-DS** [comment]
> Testing results: Components which still has following error when editing in side panel: Screenshot 2023-06-16 at 11 24 44
> 
> * progress bar - field "progress bar value"
> * navbar item in "navbar" component - all fields
> * button in "buttons" component - field "action",  "icon left", "icon left size" (same for icon right)

I fixed those in my newest commit

**Review by andrechern — APPROVED** [review]

**Pull request merged**